keyboards/dark/magnum_ergo_1/magnum_ergo_1.c

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2023 Gondolindrim <gondolindrim@acheronproject.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+void board_init(void) {
```

Comment:
Also, all of the pins should have a "set" function called. 
```suggestion
void keyboard_post_init_kb(void) {
```

Reply:
Pins set as outputs and function changed in 1c3ab2f